NewWorkbench/NewWorkbench/Content/plugins/fullcalendar/fullcalendar.css

```diff
@@ -0,0 +1,589 @@
+	   making day-numbers bold also fixes the problem */
+	}
+
+.fc-grid .fc-day-content {
+	clear: both;
+	padding: 2px 2px 1px; /* distance between events and day edges */
+	}
+
+/* event styles */
+
+.fc-grid .fc-event-time {
+	font-weight: bold;
+	}
+
+/* right-to-left */
+
+.fc-rtl .fc-grid .fc-day-number {
+	float: left;
+	}
+
+.fc-rtl .fc-grid .fc-event-time {
+	float: right;
+	}
+
+
+
+/* Agenda Week View, Agenda Day View
+------------------------------------------------------------------------*/
+
+.fc-agenda table {
+	border-collapse: separate;
+	}
+
+.fc-agenda-days th {
+	text-align: center;
+	}
+
+.fc-agenda .fc-agenda-axis {
+	width: 50px;
+	padding: 0 4px;
+	vertical-align: middle;
+	text-align: right;
+	white-space: nowrap;
+	font-weight: normal;
+	}
+
+.fc-agenda .fc-week-number {
+	font-weight: bold;
+	}
+
+.fc-agenda .fc-day-content {
+	padding: 2px 2px 1px;
+	}
+
+/* make axis border take precedence */
+
+.fc-agenda-days .fc-agenda-axis {
+	border-right-width: 1px;
+	}
+
+.fc-agenda-days .fc-col0 {
+	border-left-width: 0;
+	}
+
+/* all-day area */
+
+.fc-agenda-allday th {
+	border-width: 0 1px;
+	}
+
+.fc-agenda-allday .fc-day-content {
+	min-height: 34px; /* TODO: doesnt work well in quirksmode */
+	_height: 34px;
+	}
+
+/* divider (between all-day and slots) */
+
+.fc-agenda-divider-inner {
+	height: 2px;
+	overflow: hidden;
+	}
+
+.fc-widget-header .fc-agenda-divider-inner {
+	background: #eee;
+	}
+
+/* slot rows */
+
+.fc-agenda-slots th {
+	border-width: 1px 1px 0;
+	}
+
+.fc-agenda-slots td {
+	border-width: 1px 0 0;
+	background: none;
+	}
+
+.fc-agenda-slots td div {
+	height: 20px;
+	}
+
+.fc-agenda-slots tr.fc-slot0 th,
+.fc-agenda-slots tr.fc-slot0 td {
+	border-top-width: 0;
+	}
+
+.fc-agenda-slots tr.fc-minor th,
+.fc-agenda-slots tr.fc-minor td {
+	border-top-style: dotted;
+	}
+
+.fc-agenda-slots tr.fc-minor th.ui-widget-header {
+	*border-top-style: solid; /* doesn't work with background in IE6/7 */
+	}
+
+
+
+/* Vertical Events
+------------------------------------------------------------------------*/
+
+.fc-event-vert {
+	border-width: 0 1px;
+	}
+
+.fc-event-vert.fc-event-start {
+	border-top-width: 1px;
+	border-top-left-radius: 3px;
+	border-top-right-radius: 3px;
+	}
+
+.fc-event-vert.fc-event-end {
+	border-bottom-width: 1px;
+	border-bottom-left-radius: 3px;
+	border-bottom-right-radius: 3px;
+	}
+
+.fc-event-vert .fc-event-time {
+	white-space: nowrap;
+	font-size: 10px;
+	}
+
+.fc-event-vert .fc-event-inner {
+	position: relative;
+	z-index: 2;
+	}
+
+.fc-event-vert .fc-event-bg { /* makes the event lighter w/ a semi-transparent overlay  */
+	position: absolute;
+	z-index: 1;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	background: #fff;
+	opacity: .25;
+	filter: alpha(opacity=25);
+	}
+
+.fc .ui-draggable-dragging .fc-event-bg, /* TODO: something nicer like .fc-opacity */
+.fc-select-helper .fc-event-bg {
+	display: none\9; /* for IE6/7/8. nested opacity filters while dragging don't work */
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Update the invalid value of property "display". Expected format: inline | block | list-item | inline-block | table | inline-table | table-row-group | table-header-group | table-footer-group | table-row | table-column-group | table-column | table-cell | table-caption | none | flex | inline-flex | grid | inline-grid | run-in | contents | ruby | ruby-base | ruby-text | ruby-base-container | ruby-text-container | compact | marker [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=css%3Avalidate-property-value)